paper-tab.html

```diff
@@ -122,7 +122,8 @@
       },
 
       listeners: {
-        down: '_updateNoink'
+        down: '_updateNoink',
+        click: '_onClick'
```

Comment:
Yeah, I think `tap` is meant to group both `click`s and 'some types of click-like touch events'. But, I've always been confused as to what those special touch events are given that mobile browsers should turn them into `click` events anyways.